pygmt/src/wiggle.py

```diff
@@ -94,6 +104,26 @@ def wiggle(self, data=None, x=None, y=None, z=None, **kwargs):
     """
     kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
 
+    if (fill_positive or fill_negative) and kwargs.get("G") is not None:
+        raise GMTInvalidInput("Use either fill_positive/fill_negative or color.")
+
+    if kwargs.get("G") is not None:
+        msg = (
+            "The 'color' parameter has been deprecated since v0.8.0"
+            " and will be removed in v0.12.0. Use fill_positive/fill_negative"
+            " instead."
+        )
+        warnings.warn(msg, category=FutureWarning, stacklevel=2)
+
+    if fill_positive:
+        kwargs["G"] = fill_positive + "+p"
+
+    if fill_negative:
+        if kwargs.get("G") is not None:
+            kwargs["G"] = [kwargs["G"], fill_negative + "+n"]
+        else:
+            kwargs["G"] = fill_negative + "+n"
```

Comment:
The following codes are simpler:
```suggestion
    if fill_positive or fill_negative:
        kwargs["G"] = []
        if fill_positive:
            kwargs["G"].append(fill_positive + "+p")
        if fill_negative:
            kwargs["G"].append(fill_negative + "+n")
```